docs/components_and_bundles/bundles/SyliusGridBundle/configuration.rst

```diff
@@ -54,13 +54,14 @@ Here you will find all configuration options of ``sylius_grid``.
                         form_options: { }
                         default_value: ~
                     date:
-                        type: date # Type of filter
+                        type: datetime # Type of filter
```

Comment:
Are you sure it does not work? As far as I see, it's not a field type but filter type (and it exists https://github.com/Sylius/SyliusGridBundle/blob/1.4/src/Component/Filter/DateFilter.php)

Reply:
I posted the exception I get when using the `date` type.

Reply:
Oh god my mistake.